src/components/landing/landing.js

```diff
@@ -19,115 +19,76 @@ class Landing extends Component {
-                    </button>
-                  )}
-                />
-              </div>
-            </AuthComponent>
-          </article>
-          {events.length &&
-            events.map(block => this.renderCarouselBlock(block))}
-          {/* <article className="events-article">
-            <div className="row">
-              <div className="col-md-10">
-                <h3>Upcoming events</h3>
-              </div>
-              <div style={{ textAlign: "right" }} className="col-md-2">
-                <a href="/home/events">Explore more events</a>
-                <EventMatrix />
+            <div>
+              <i className="fas fa-search"></i>
+              <h3>Find Events Partners</h3>
+            </div>
+            <div>
+              <i className="far fa-smile"></i>
+              <h3>Get Connected</h3>
+            </div>
+            <div>
+              <i className="fas fa-users"></i>
+              <h3>Have Fun Together</h3>
+            </div>
+          </div>
+          <AuthComponent onlyAnonymous>
+            <div className="text-center">
+              <ModalWind
+                renderButton={action => (
+                  <button
+                    className="btn btn-warning"
+                    onClick={() => action()}
+                    type="button"
+                  >
+                    Join EventsExpress
+                  </button>
+                )}
+              />
+            </div>
+          </AuthComponent>
+        </article>
+        {items.length !== 0 && (
+          <>
+            <article className="events-article">
+              <div className="row">
+                <div className="col-md-10">
+                  <h3>Upcoming events</h3>
+                </div>
               </div>
+            </article>
+            <div className="container">
+              <SliderForEvents events={items} />
+            </div>
+            <div className="container">
+              {items.length !== 0 && <EventMatrix events={items} />}
             </div>
-            <div className="carousel-wrapper text-center">
-              <Carousel
-                autoPlay={false}
-                animation="slide"
-                interval={1000}
-                indicators={false}
-                navButtonsAlwaysVisible={carouselNavIsVisible}
-                navButtonsAlwaysInvisible={!carouselNavIsVisible}
-                NextIcon={
-                  <i
-                    style={{ width: `${32}px`, height: `${32}px` }}
-                    className="fas fa-angle-right"
-                  ></i>
-                }
-                PrevIcon={
-                  <i
-                    style={{ width: `${32}px`, height: `${32}px` }}
-                    className="fas fa-angle-left"
-                  ></i>
-                }
-              >
-                {events.map(block => this.renderCarouselBlock(block))} 
-              </Carousel>
+            <div className="container">
```

Comment:
Same

Reply:
Yes, but I'm already working on would be one container. That will be in the next PR.